app/models/maintenance_tasks/task_data.rb

```diff
@@ -157,6 +181,11 @@ def new
       MaintenanceTasks::Task.named(name).new
     end
 
+    # @return [Boolean] whether the Task has any Run.
+    def has_run?
```

Comment:
Let's make this private?

Reply:
This method is called by `.find` on line 31, so making it private would not work. Unless you're suggesting we do `send(:has_run?)` there?